cmd/dlt/dlt.go

```diff
@@ -0,0 +1,98 @@
+package dlt
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+// InstallDLTSymlink creates a symlink named 'dlt' pointing to the real databricks binary.
+func InstallDLTSymlink() error {
+	path, err := exec.LookPath("databricks")
+	if err != nil {
+		return errors.New("databricks CLI not found in PATH")
+	}
+	realPath, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return fmt.Errorf("failed to resolve symlink: %w", err)
+	}
+
+	dir := filepath.Dir(path)
+
+	// Check write permission by trying to create and remove a temp file
+	testFile := filepath.Join(dir, ".permtest")
```

Comment:
Not sure if we need to do this, we can just rely on `os.Symlink` fail to create symlink with whatever error is, it will likely included details about not having permissions